autofs-5.1.1 - fix fix gcc5 complaints

From: Ian Kent <raven@themaw.net>

Remove a stray declare of dump_core().

Signed-off-by: Ian Kent <raven@themaw.net>
---
 lib/rpc_subs.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index 846c40e..84fae9e 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -77,7 +77,6 @@ static const rpcvers_t mount_vers[] = {
 };
 
 static int connect_nb(int, struct sockaddr *, socklen_t, struct timeval *);
-inline void dump_core(void);
 
 /*
  *  Perform a non-blocking connect on the socket fd.
